### PR TITLE
Hotfix/ad report model

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
**Are you a current Fivetran customer?** 

Fivetran employee

**What change(s) does this PR introduce?** 

Updates ad report model to get ad id from ad stats rather than ad history

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 
This is a hotfix to unblock the release of https://github.com/fivetran/dbt_ad_reporting/issues/69- temporarily fixes failing integration tests for 🧱 Databricks 🧱

**How did you test the PR changes?** 

 

- [ ] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: .buildkite/scripts/run_models.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
```
Comment (fivetran-avinash) [inline .buildkite/scripts/run_models.sh:3]: Same question as other `run_models.sh` changes.

File: integration_tests/ci/sample.profiles.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 
-# HEY! This file is used in the Google Ads integrations tests with CircleCI.
+# HEY! This file is used in the dbt package integrations tests with CircleCI.
```
Comment (fivetran-avinash) [inline integration_tests/ci/sample.profiles.yml:2]: Change to Buildkite? Or maybe it'll get batch updated later? 😄

Comment (fivetran-avinash) [reply integration_tests/ci/sample.profiles.yml:2]: Also looks as if `.github/pull_request_template.md` wasn't changed.

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/google_ads_source
-    version: [">=0.8.0", "<0.9.0"]
+  # - package: fivetran/google_ads_source
```
Comment (fivetran-avinash) [inline packages.yml:2]: does this need to be uncommented  and the current package removed?

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-sheringuyen Looks good! Few cosmetic changes left 🖊️

File: .buildkite/scripts/run_models.sh (hunk 1 above)
Comment (fivetran-sheringuyen) [reply .buildkite/scripts/run_models.sh:3]: Refer to other replies

File: integration_tests/ci/sample.profiles.yml (hunk 2 above)
Comment (fivetran-sheringuyen) [reply integration_tests/ci/sample.profiles.yml:2]: q1: will be batched later
q2: good catch, updating

File: packages.yml (hunk 3 above)
Comment (fivetran-sheringuyen) [reply packages.yml:2]: yep! will have to swap this before official release of this pkg, but need this in order for pkg to pass Buildkite in the meantime!

Comment (fivetran-avinash) [reply packages.yml:2]: Ahhh yep!

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-sheringuyen Approved!

**Pull request merged**